core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -302,12 +302,12 @@ class DAGScheduler(
       shuffleDep: ShuffleDependency[_, _, _],
       firstJobId: Int): ShuffleMapStage = {
     val rdd = shuffleDep.rdd
-    val numTasks = rdd.partitions.size
+    val numTasks = rdd.partitions.length
```

Comment:
super nit, but any reason for all the `size` -> `length` changes?  just seems like a bit of noise if we ever look in git history for these lines.